openelis/src/MessageResources.properties

```diff
@@ -2919,6 +2919,8 @@ dashboard.sample.column.link.collectSample = Collect Sample
 dashboard.sample.column.link.result = Result
 dashboard.sample.column.link.validate = Validate
 dashboard.sample.column.link.print = Print
+dashboard.Main.Contact = Main Contact

```

Comment:
Also remove these `^M` characters

Reply:
^M is also reflected on Master branch of OpenElis , In local this not is reflected I think this By default. Please have look on master Repo.